_guides/responsive/control_layout.md

```diff
@@ -1,45 +1,26 @@
 ---
 layout: page
-title: How to Control Layout
+title: Supported Layouts
 order: 1
 ---
-To control layout,
-define the size attributes and layout behavior for all AMP elements.
-As with all web pages, use media queries
-to control the page layout for different screen sizes.
-For more control over element sizing, consider using `size` and `srcset` attributes.
-Include a placeholder so that users see something before the page loads.
-Also provide fallbacks for any page element
-in case the user’s browser doesn’t support it.
 
-{% include toc.html %}
+AMP provides a set of layouts that AMP elements can use.
+The recommended `layout` attribute specifies
+how the component behaves in the document layout.
 
-## How AMP lays out its pages
+In addition to these specific layout types,
+you can also use CSS media queries,
+`srcset` and `sizes` attributes, as you would with any website.
 
-The AMP layout system’s key goal is to ensure that every element can be displayed
-and its space properly preserved before any of the JavaScript
-or remote resources have been downloaded.
-This significantly reduces rendering and scrolling jank.
-To express each element’s size and position on a page,
-the layout system relies on these attributes:
-`width`, `height`, `layout`, `media`, `sizes`, `placeholder`, `fallback`.
+The `placeholder`, `fallback`, and `noloading` attributes
```

Comment:
There are other layout types though-- presumably you mean just the intro? Or do you mean there's no need to describe the other supported layout types all together?